css/style.scss

```diff
@@ -1072,6 +1070,25 @@ input[type="password"] {
 /**
  * Dark-Theme fixes
  */
+.avatar.icon {
+	// We always want to use the white icons, this is why we don't use var(--color-white) here.
+	&.icon-public {
+		background-image: url(icon-color-path('public', 'actions', 'fff', 1, true));
```

Comment:
@skjnldsv @juliushaertl so just you know, that those methods are considered public API and are used by apps.

Reply:
Would it be better to use `@include icon-color('public', 'actions', 'fff', 1, true)` instead?